frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,182 @@
+            text="bundle-feature-one"
+            position="feature-one"
+            vars={{
+              num_vpn_servers: "400",
+            }}
+          />
+          <FloatingFeatures
+            icon={bundleFloatTwo.src}
+            text="bundle-feature-two"
+            position="feature-two"
+            vars={{
+              num_vpn_countries: "30",
+            }}
+          />
+          <FloatingFeatures
+            icon={bundleFloatThree.src}
+            text="bundle-feature-three"
+            position="feature-three"
+          />
+        </div>
+      </div>
+      {isFlagActive(props.runtimeData, "bundle") &&
+        isBundleAvailableInCountry(props.runtimeData) && (
+          <div className={styles["second-section"]}>
+            <div className={styles["bundle-banner-description"]}>
+              <h2>
+                <Localized
+                  id={"bundle-banner-header"}
+                  elems={{
+                    "vpn-logo": <VpnWordmark />,
+                  }}
+                >
+                  <span className={styles["headline"]} />
+                </Localized>
+              </h2>
+              <h3>{l10n.getString("bundle-banner-subheader")}</h3>
+              <p>{l10n.getString("bundle-banner-body")}</p>
+              <Localized
+                id={"bundle-banner-1-year-plan"}
+                elems={{
+                  b: <b />,
+                }}
+              >
+                <p />
+              </Localized>
+              <div className={styles["pricing-logo-wrapper"]}>
+                <div className={styles["pricing-wrapper"]}>
+                  <Localized
+                    id={"bundle-price-monthly"}
+                    vars={{
+                      monthly_price: getBundlePrice(props.runtimeData),
+                    }}
+                    elems={{
+                      "monthly-price": <p className={styles["price"]} />,
+                    }}
+                  >
+                    <span />
+                  </Localized>
+                  <Localized
+                    id={"bundle-price-save-amount"}
+                    vars={{
+                      savings: "??%", // Design states 50%
+                      old_price: "$??", // Design states $11.99
+                    }}
+                    elems={{
+                      "outdated-price": <s className={styles["price"]} />,
+                    }}
+                  >
+                    <span />
+                  </Localized>
+                </div>
+                <img src={bundleLogo.src} alt="Bundle logo" />
+              </div>
+              <div className={styles["bottom-section"]}>
+                <LinkButton
+                  target="_blank"
+                  className={styles["button"]}
+                  href={getBundleSubscribeLink(props.runtimeData)}
+                  onClick={() =>
+                    trackPurchaseStart({ label: "bundle-banner-upgrade-promo" })
```

Comment:
Hmm OK, this is what's annoying about working on interconnected things at the same time, but in https://github.com/mozilla/fx-private-relay/pull/2471 I added `trackPlanPurchaseStart` to make sure we differentiate between the different plans. I think/hope that maybe you could merge that branch in without conflicts and then use that?

```suggestion
                    trackPlanPurchaseStart({ plan: "bunde" }, { label: "bundle-banner-upgrade-promo" })
```

Reply:
Oh, and maybe we also want to add a `useGaViewPing` to this so we can track how often people actually see this button before clicking on it :)

Reply:
Ah, I'm a little scared of the merge conflicts. Can I open a follow up PR to fix that?

Reply:
I added a to-do comment above it.

Reply:
I added it in 68429074 after bringing it in line with current `main`.